.gitignore

```diff
@@ -110,3 +110,8 @@ ENV/
 
 # mypy
 .mypy_cache/
+.pydevproject
```

Comment:
I'm unfamiliar with these enteries. Do they belong in a project level ignore or more in a local/global configuration?

Reply:
Yes,is my local configuration,sorry!I'm use eclipse and pydev for develop.